README.rst

```diff
@@ -1,6 +1,7 @@
 =========
 Spidermon
 =========
+The battle tested spider monitoring library made with 💝 by `ScrapingHub <https://blog.scrapinghub.com/spidermon-scrapy-spider-monitoring>`_ is now open-source.
```

Comment:
I don’t think adding this line is a good idea, and the intent of this line conflicts with the introductory paragraph of the Overview section, and some of information this line adds seems a bit unnecessary to me:
- ‘battle-tested’ sounds a bit ‘propagandistic’ to me
- While Scrapinghub supports the project, I am unsure of whether this should be mentioned in the README file
- ‘… is now …’ is a phrasing for an announcement made at a specific point in time. I don’t think the description of Spidermon should include that.

I wouldn’t mind simplifying the existing overview text, or removing the Overview header that precedes it (the first paragraph of a document is always an overview, no need for a header). But I don’t think adding a new line here is the right approach.

Reply:
Thanks, the entire line does sound _propagandistic_ to me as well. I just thought it would be a good idea to add that since I picked it up directly from the [ScrapingHub blog](https://blog.scrapinghub.com/spidermon-scrapy-spider-monitoring). Which caught a lot of attention in the news. The people who read the article then visit this repo, might see a stroke of relation. On second thought, we should keep it simple and not too glamorous as to say.

> ‘… is now …’ is a phrasing for an announcement made at a specific point in time. I don’t think the description of Spidermon should include that.

Picked it up from the blog and now that I realize, Spidermon has been out for quite some time now. My fault that one. 

 I agree with you all the way, will remove this line.
(Learned something new here)

Reply:
Does Spidermon have a logo? Logos look well in repository README's.
Might be a good if we are going for representation and formatting. Just thinking out loud.

Reply:
Sadly, I don’t think we have a logo at this point.

Reply:
Cropped it from the ScrapingHub blog about Spidermon. I am not sure if this is it, or what. Might try running it by people on Slack. ![Screenshot_2019-03-22_19-52-00](https://user-images.githubusercontent.com/22801822/56281686-e204d900-612a-11e9-9a65-1deb2133da31.png)